site/content/en/docs/manual/advanced/cli.md

```diff
@@ -21,7 +22,24 @@ Overview of functionality:
 - Export and download a whole task
 - Import a task
 
-**Usage**
+CVAT Command line interface use `argparse`.
+For learn more read the [argparse documentation](https://docs.python.org/library/argparse.html#).
+
+## Usage
+
+If you want to access the console remotely via `ssh`,
```

Comment:
@TOsmanov , I don't think that the information is relevant for CLI. Could you please remove all info about ssh?